app/src/main/res/layout/walkthrough_topic_summary_view.xml

```diff
@@ -13,24 +13,27 @@
     android:id="@+id/walkthrough_topic_container"
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
+    android:layout_margin="@dimen/walkthrough_topic_summary_container_margin"
     android:clickable="true"
     android:elevation="4dp"
-    android:layout_margin="@dimen/walkthrough_topic_summary_container_margin"
     android:focusable="true"
     android:onClick="@{(v) -> viewModel.clickOnSummaryTile()}"
     app:roundedRectDrawableWithColor="@{viewModel.backgroundColor}">
 
-    <ImageView
+    <org.oppia.android.app.customview.LessonThumbnailImageView
       android:id="@+id/walkthrough_topic_thumbnail_image_view"
       android:layout_width="match_parent"
       android:layout_height="92dp"
       android:importantForAccessibility="no"
       android:scaleType="centerInside"
-      android:src="@{viewModel.topicSummary.topicThumbnail.thumbnailGraphic}"
+      app:entityId="@{viewModel.topicSummary.topicId}"
+      app:entityType="@{viewModel.topicEntityType}"
       app:layout_constraintBottom_toTopOf="@+id/bottom_container"
+      app:layout_constraintDimensionRatio="4:3"
```

Comment:
Is this always correct, or does it depend on the image?

Reply:
I used this from `topic_summary_view`

Reply:
@BenHenning I took `topic_summary_view` as reference for this 
https://github.com/oppia/oppia-android/blob/022b4c6d242d28e5c8dc2e030abef8b16f2b55c6/app/src/main/res/layout/topic_summary_view.xml#L33

Reply:
@BenHenning This ratio was finalised after discussion with @mschanteltc

Reply:
Ack--thanks for the context @rt4914!